backend/Dockerfile

```diff
@@ -1,4 +1,19 @@
-FROM public.ecr.aws/docker/library/node:16-alpine
+FROM public.ecr.aws/docker/library/node:18-alpine as builder
+WORKDIR /usr/home/postmangovsg
+
+COPY shared ../shared
+RUN cd ../shared && npm ci
+
+COPY ./package* ./
+RUN npm ci
+
+COPY tsconfig.* ./
```

Comment:
trivial, but `tsconfig` should come before the others, since it's less likely to change?